markdown/chapters/meandeck/deckconstruction.md

```diff
@@ -0,0 +1,603 @@
+- 3 Daze
+- 1 Thoughtseize
+- 2 Duress
+- 4 Baleful Strix
+- 3 Uro, Titan of Nature's Wrath
+
+This list has invested in having the alternate gameplan as part of it's construction. 
+Free cycle effects and Lotus Petals will probably both be very low, land count will
+likely be high and you can see they have trimmed on a Daze here too. This list
+will still be able to win fast but is expecting to have a longer and grindier
+game most likely.
+
+## 5. Card Group - Cantrips
+
+<row variant="pile">{{!BS}} {{!Pnd}} {{!Pre}} {{!Mishra's Bauble}}</row>
+
+Cantrips are a staple in {U} decks in Legacy and provide the ability to find the
+cards you need at the time you need them and allow you to *cheat* on mana resources
+as you don't have to rely on the top of your deck or openening hand to draw them.
+I won't write too much in detail on the different cantrips used however if
+you wish to read more about the ways of using them in the deck, 
+feel free to read [this article here](/articles/2020/10/05/gangsta-trippin/). 
+
+There are two core cantrips which are run as a core part of the
+deck package; {{Brainstorm}} and {{Ponder}}. These are both run as four-ofs and, 
+without very good justification, should likely never drop below that number.
+Preordain is a more flexible cantrip, normally being run as at least a two-of
+but anywhere up to a four-of most of the time. As said before the number of
+Preordains tends to fluctuate with the style of the deck often being cut in
+favour of tutor effects or creature packages.
+
+{{Mishra's Bauble}} is a card that generally only gets played in slower builds
+alongside high numbers of {{Predict}}. It acts as a delayed way to draw into
+a pile as well as providing fetching information for either your own deck or
+in terms of what your opponent is playing. Most importantly though it allows
+you to consistently *hit* with Predict allowing the maximum card advantage to
+be accrued with it. Outside of this interaction though, it is likely worse
+than Preordain despite the lower cost as it decreases your potential blue
+count for Force of Will and can be slow/not contribute to same turn wins
+when you need them.
+
+### 5.1 Deckbuilding Ratios - Cantrips
+
+Most lists run around eleven cantrip effects, not including free cycle effects
+or other pile cards like Predict. This is generally a 4:4:3 split of
+Brainstorm:Ponder:Preordain with the Preordain slot being the one that changes.
+It's a very flexible slot and can be replaced by any number of things including
+interaction, free cycle effects or tutor effects. If you are unsure on what
+style of list you wish to play it is likely better to run the full twelve
+cantrips and then work out what you wish to replace them with at a later point
+in testing.
+
+## 6. Card Group - Pile Cards
+
+
+### 6.1 Thassa's Oracle
+
+This is probably the most diverse set of cards and might as well be titled 
+"miscellaneous" however there are some important pieces in here that 
```

Comment:
Maybe we can use this as an introductory sentence and move it from 6.1 to 6

Reply:
Amended.